libc/lib/CMakeLists.txt

```diff
@@ -1,12 +1,9 @@
 set(libc_archive_targets "")
+  list(APPEND added_archive_targets ${archive_1})
```

Comment:
I don't think this is in the right place since `archive_1` is defined below.

Reply:
Don't know how that got there, I'll fix it.